modules/pxp-module-puppet

```diff
@@ -109,10 +110,10 @@ def get_result_from_report(exitcode, config, start_time)
   return run_result
 end
 
-def start_run(config, params)
+def start_run(config, action_input)
```

Comment:
If it's appropriate to rename `params` -> `action_input` here, isn't it also appropriate to rename them at the point of use?  https://github.com/mruzicka/pxp-agent/blob/PCP-251_simple_output_files_support/modules/pxp-module-puppet#L64

I don't think the rename adds much, but if done it should be consistent.